requirements.txt

```diff
@@ -1,15 +1,15 @@
+# This file is also processed by conda-build, so keep spaces as "pkg >=1.2"
+appdirs
+dask >=0.17.0
+holoviews
+ipywidgets >=7.2
 jinja2
 msgpack-numpy
 msgpack-python
 numpy
 pandas
-pytest
 python-snappy
-ruamel.yaml >= 0.15.0
 requests
-appdirs
+ruamel.yaml >=0.15.0
 six
-tornado >= 4.5.1
-dask[complete]
-holoviews
-ipywidgets >= 7.2
+tornado >=4.5.1
```

Comment:
I suppose this is strictly optional too - the server won't work without it, but everything else will. Let's defer for now, it would require at least a good message on failure in the server main.